packages/website/plugins/generated-rule-docs.ts

```diff
@@ -40,22 +46,228 @@ const generatedRuleDocs: Plugin = () => {
+            url: `https://eslint.org/docs/rules/${file.stem}#options`,
+          },
+          {
+            type: 'text',
+            value: '.',
+          },
+        ],
+        type: 'paragraph',
+      } as mdast.Paragraph);
+
+      parent.children.splice(howToUseH2Index + 1, 0, {
+        lang: 'js',
+        type: 'code',
+        meta: 'title=".eslintrc.cjs"',
+        value: `module.exports = {
+  // Note: you must disable the base rule as it can report incorrect errors
+  "${file.stem}": "off",
+  "@typescript-eslint/${file.stem}": "${optionLevel}"
+};`,
+      } as mdast.Code);
+    } else {
+      parent.children.splice(optionsH2Index, 0, {
+        lang: 'js',
+        type: 'code',
+        meta: 'title=".eslintrc.cjs"',
+        value: `module.exports = {
+  "rules": {
+    "@typescript-eslint/${file.stem}": "${optionLevel}"
+  }
+};`,
+      } as mdast.Code);
+
+      if (meta.schema.length === 0) {
+        parent.children.splice(optionsH2Index + 1, 0, {
+          children: [
+            {
+              type: 'text',
+              value: 'This rule is not configurable.',
+            },
+          ],
+          type: 'paragraph',
+        } as mdast.Paragraph);
+      } else {
+        const optionsSchema =
+          meta.schema instanceof Array ? meta.schema[0] : meta.schema;
+
+        parent.children.splice(
+          optionsH2Index + 2,
+          0,
+          {
+            children: [
+              {
+                type: 'text',
+                value: `This rule accepts an options ${
+                  'enum' in optionsSchema
+                    ? 'string of the following possible values'
+                    : 'object with the following properties'
+                }:`,
+              } as mdast.Text,
+            ],
+            type: 'paragraph',
+          } as mdast.Paragraph,
+          {
+            lang: 'ts',
+            type: 'code',
+            value: [
+              (
+                await compile(
+                  {
+                    title: `Options`,
+                    ...optionsSchema,
+                  },
+                  file.stem,
+                  {
+                    additionalProperties: false,
+                    bannerComment: '',
+                    declareExternallyReferenced: true,
+                  },
+                )
+              ).replace(/^export /gm, ''),
+              format(
+                `const defaultOptions: Options = ${JSON.stringify(
+                  rule.defaultOptions,
+                )};`,
+                {
+                  parser: tseslintParser.parse,
+                },
+              ),
+            ]
+              .join(EOL)
+              .trim(),
+          } as mdast.Code,
+        );
+      }
+    }
+
+    // 6. Add a notice about coming from ESLint core for extension rules
+    if (meta.docs.extendsBaseRule) {
+      parent.children.push({
+        children: [
+          {
+            type: 'text',
```

Comment:
Hi hi

If you want to make `<sup>` work, you have to make it a separate child node with `type: "jsx"`. MD inside JSX won't be interpolated